packages/components/src/avatar/tests/chromatic/AvatarGroup.chroma.jsx

```diff
@@ -1,205 +1,228 @@
+            <Avatar name="Neil Armstrong" />
+        </AvatarGroup>
+    </Stack>;
+
+export const Overflow = () =>
+    <Div width={10}>
+        <AvatarGroup size="2xl">
+            <Avatar name="Sally Ride" />
+            <Avatar name="Alan Shepard" />
+            <Avatar name="Chris Hadfield" />
+            <Avatar name="Christa McAuliffe" />
+        </AvatarGroup>
+    </Div>;
+
+export const Zoom = () =>
+    <Stack>
+        <Div className="zoom-in">
+            <AvatarGroup>
                 <Avatar name="Sally Ride" />
                 <Avatar name="Alan Shepard" />
-                <Avatar name="Chris Hadfield" />
-                <Avatar name="Christa McAuliffe" />
             </AvatarGroup>
         </Div>
-    )
-    .add("zoom", () =>
-        <Stack>
-            <Div className="zoom-in">
-                <AvatarGroup>
-                    <Avatar name="Sally Ride" />
-                    <Avatar name="Alan Shepard" />
-                </AvatarGroup>
-            </Div>
-            <Div className="zoom-out">
-                <AvatarGroup>
-                    <Avatar name="Sally Ride" />
-                    <Avatar name="Alan Shepard" />
-                </AvatarGroup>
-            </Div>
-        </Stack>
-    )
-    .add("styling", () =>
-        <Inline>
-            <AvatarGroup border="warning-7">
-                <Avatar name="Sally Ride" />
-                <Avatar name="Alan Shepard" />
-            </AvatarGroup>
-            <AvatarGroup className="border-red">
-                <Avatar name="Sally Ride" />
-                <Avatar name="Alan Shepard" />
-            </AvatarGroup>
-            <AvatarGroup style={{ border: "1px solid red" }}>
+        <Div className="zoom-out">
+            <AvatarGroup>
                 <Avatar name="Sally Ride" />
                 <Avatar name="Alan Shepard" />
             </AvatarGroup>
-        </Inline>
-    );
+        </Div>
+    </Stack>;
+
+export const Styling = () =>
+    <Inline>
+        <AvatarGroup border="warning-7">
+            <Avatar name="Sally Ride" />
+            <Avatar name="Alan Shepard" />
+        </AvatarGroup>
+        <AvatarGroup className="border-red">
+            <Avatar name="Sally Ride" />
+            <Avatar name="Alan Shepard" />
+        </AvatarGroup>
+        <AvatarGroup style={{ border: "1px solid red" }}>
+            <Avatar name="Sally Ride" />
+            <Avatar name="Alan Shepard" />
+        </AvatarGroup>
+    </Inline>;
+
+export const WithRemainingsHover = () =>
+    <AvatarGroup remainingAvatarsProps={{ "data-testid": "remaining-avatars" }}>
+        <Avatar name="Sally Ride" />
+        <Avatar name="Alan Shepard" />
+        <Avatar name="Chris Hadfield" />
+        <Avatar name="Christa McAuliffe" />
+        <Avatar name="Neil Armstrong" />
+    </AvatarGroup>
+;
+
+WithRemainingsHover.play = async () => {
+    userEvent.hover(screen.getByTestId("remaining-avatars"));
+};
+
+Default.storyName = "default";
```

Comment:
why are those storyName stacked at the bottom instead of directly after the elements. All the files before this one write the storyName right after the component

Reply:
This is an ommission